.env.example

```
@@ -54,3 +54,14 @@ ROUTINE_ENRICH_MODEL="claude-opus-4-8"
 OPENAI_API_KEY="sk-openai-change-me"
 OPENAI_IMAGE_MODEL="dall-e-3"
 
+# Auto-publish LinkedIn cross-post (spec §2.5, Phase 6). When both
+# LINKEDIN_ACCESS_TOKEN and LINKEDIN_AUTHOR_URN are set, publishing a draft
+# best-effort shares the post to LinkedIn (post-commit; a failure never reverts
+# the publication). LINKEDIN_AUTHOR_URN is the person/org URN
+# (e.g. urn:li:person:xxxx); SITE_URL is the public origin used to build the
+# canonical post link. Leave the LinkedIn vars unset to skip cross-posting.
+LINKEDIN_ACCESS_TOKEN="change-me-linkedin-token"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)